cdap-master/src/main/java/co/cask/cdap/data/tools/Main.java

```diff
@@ -157,119 +266,107 @@ private Action parseAction(String action) {
     }
   }
 
-  private void performUpgrade(Injector injector) throws Exception {
-    // Upgrade system datasets
-    upgradeSystemDatasets(injector);
+  private void performUpgrade() throws Exception {
+    LOG.debug("Upgrading System and User Datasets ...");
```

Comment:
May be a good idea to have these logs at ``INFO`` level, since this is a tool that people may want to monitor progress closely?

I would make them INFO atleast until we finish testing.